staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -280,6 +280,18 @@ var (
 		[]string{"code_path"},
 	)
 
+	WatchListLatencies = compbasemetrics.NewHistogramVec(
+		&compbasemetrics.HistogramOpts{
+			Subsystem: APIServerComponent,
+			Name:      "watch_list_duration_seconds",
+			Help:      "Response latency distribution in seconds for watch list requests broken by group, version, resource and scope.",
+			// TODO(p0lyn0mial): adjust the buckets
+			Buckets:        []float64{0.5, 1.0, 2, 5, 8, 10, 15, 20, 30, 45, 60},
```

Comment:
> Agree - maybe we don't need all the buckets from:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go#L99

> but I would definitely start with something lower (0.025?) and just took a subset of those

@wojtek-t I think that 0.025 is 25 milliseconds, which seems to be too low, no ? did you mean `0.25` :) ?

Reply:
I meant 0.025 - for very small lists it's not unreasonable. Maybe 0.05 - but I want to see some percent of those to be below 100ms eventually (once Marek's improvements with manual progress notify land).

Reply:
>  below 100ms eventually

wow, nice :) I can add 0.05 in addition to what we have atm.

can you explain what Marek will be working on ?